### PR TITLE
patch i guess

### PR DESCRIPTION
**Pull request by PowerMacFan**
read the differences i guess

### CODE REVIEW

File: PyPrograms/equations/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+@bupboi1337
+
+## What does it do?
+
+well... equations I think?
+
+## How to install
+
+run it with python
```
Comment (PowerMacFan) [inline PyPrograms/equations/readme.md:12]: could explain other stuff but i wanted to keep it simple

File: PyPrograms/equations/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Equations
+A python program Orgiginally created by @JymPatel and edited by:
+
+@bupboi1337
+
+## What does it do?
+
+well... equations I think?
```
Comment (PowerMacFan) [inline PyPrograms/equations/readme.md:8]: i just wanted to keep it basic but i can improve on it later

**Review by PowerMacFan — COMMENTED** [review]
I think this would be a very good patch to approve. I would recommend pushing it whenever you want.

**Comment by jayam04** [comment]
Thanks! For being here. Hope we create the best community.

**Pull request merged**